src/rc/start-stop-daemon.c

```diff
@@ -1006,6 +1025,36 @@ int main(int argc, char **argv)
 		for (i = getdtablesize() - 1; i >= 3; --i)
 			close(i);
 
+		if (scheduler != NULL) {
+			int scheduler_index;
+			struct sched_param sched =  {.sched_priority = sched_prio};
+			if (strcmp(scheduler, "fifo") == 0)
+				scheduler_index = SCHED_FIFO;
+			else if (strcmp(scheduler, "rr") == 0)
+				scheduler_index = SCHED_RR;
+			else if (strcmp(scheduler, "other") == 0)
+				scheduler_index = SCHED_OTHER;
+#ifdef __linux__
```

Comment:
would prob be better to use `#ifdef SCHED_BATCH` for each subsection rather than tie it to the OS